task09-reactjs/my-app/src/components/Main.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import CardGrid from './CardGrid';
+import MainSection from './MainSection';
+
+class Main extends React.Component {
+
+  constructor() {
+    super();
+    this.state = {
+      islands : [],
+      characters : [],
+      mysticObjects : []
+    };
+    this.URL = 'https://static.akurey.com/trainings/OnePieceInformation.json';
+  }
+
+  componentDidMount () {
+    fetch( this.URL )
+    .then( response => response.json() )
+    .then( response => {
+      response.characters.sort(this.sortByName);
+      response.islands.sort(this.sortByName);
+      response.mysticObjects.sort(this.sortByName);
+      this.setState(response);
+    })
+    .catch((e) => console.log('Something went wrong while fetching data from '+this.URL+'. See error below:\n'+ e))
```

Comment:
TIP: console has other ways to debug messages, for errors you could use `console.error(...)` and there is also `console.warn(...)`, just in case you need it. :)